static/styles/styles.css

```diff
@@ -73,4 +73,104 @@ form {
   text-align: center;
   font-size: 16px;
   transition: all 0.5s ease;
+}
+
+/* prayer-button styling */
```

Comment:
I ran the animation locally and it looks great. however, is there an animation we can leverage that doesn't require so much code? There's a lot going on here. Is there a simpler solution? We just need some indication that it has been clicked. Doesn't need to be anything fancy. Something like this could work: https://codepen.io/ash_s_west/pen/GRZbvym

But feel free to find a better, simpler option. Also, might not be a bad idea to disable the button after it's been clicked and provide some sort of UI indicator that is has been clicked and cannot be clicked again. Let me know what you think.

Reply:
I agree, I thought it was too much too. I will do my search and send you a couple of examples, also I like the one you sent, so if that's good with you, I believe it matches more the theme of the page.
Thanks for the feedback.

Reply:
This is better, but I think we need to grab the user's attention a little bit more. Perhaps share some options of what you might thinking

Reply:
Sounds good, I'm working on it and will show real soon.